src/store.js

```diff
@@ -269,11 +295,18 @@ class Store {
 
   /**
    * @private
+   * @returns {Object}
    */
   fetchUnknown (link, query) {
     return this._fetch(link, query)
   }
 
+  /**
+   * Creates an internal-model that will be stored in the Store's
+   * cache.
+   * @param {internal-model} resource
+   * @returns {void}
```

Comment:
`{void}`? I thought this returns `{Promise<ModelProxy>}` ?

Reply:
Good call, you're right! I usually view "create" methods as methods that produce side effects more than returning something. Will update.